internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -29,6 +30,15 @@ var (
 	errInvalidUIDGIDConfig          = errors.New("must specify both UID and GID, or neither")
 	errInvalidEFSConfig             = errors.New("bad EFS configuration: cannot specify both bool and config")
 	errReservedUID                  = errors.New("UID must not be 0")
+	errCircularDependency           = errors.New("Bad dependency name: circular dependency present")
```

Comment:
nit: in golang, error strings conventionally begin with a lower case letter

Reply:
Adjusted to lowercase